orangecontrib/bioinformatics/annotation/annotate_samples.py

```diff
@@ -92,23 +94,19 @@ def _cpm(data):
         return data / np.sum(data, axis=1)[:, None] * 1e6
 
     @staticmethod
-    def select_attributes(data, z_threshold=1):
+    def mann_whithney_test(data):
```

Comment:
Typo here. :) It's not major, but probably function names should be written correctly.

Reply:
Thank you @ajdapretnar it is changed.